fix(tooltip): handle body removal

Theeraphat-Sorasetsakul opened the pull request:
## Description

Handle the removal with body existence check. Optional chaining e.g. `document.body?.<prop-name>` is quite handy for this.

Fixes # ([DME-2103](https://jira.refinitiv.com/browse/DME-2103))

**Tested:**
After remove body, checked at memory tab on devtool that there is no EventListener called from tooltip-manager.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

Commit:
fix(tooltip): handle body removal

Commit:
revert(overlay): revert change

wattachai-lseg commented on packages/elements/src/overlay/managers/interaction-lock-manager.ts:
```diff
@@ -162,12 +162,12 @@ export class ScrollLockManager {
     if (document.scrollingElement) {
-      this.scrollTop = document.scrollingElement.scrollTop;
-      this.scrollLeft = document.scrollingElement.scrollLeft;
+      this.scrollTop = document.scrollingElement?.scrollTop || 0;
+      this.scrollLeft = document.scrollingElement?.scrollLeft || 0;
```
let's revert this one similar to v7 PR.

Theeraphat-Sorasetsakul replied:
Fixed.

Commit:
revert(overlay): revert change

refinitiv-ui-dev committed:
chore(release): publish [skip ci]

 - @refinitiv-ui/demo-block@6.1.23-next.0
 - @refinitiv-ui/elements@6.16.3-next.0

Commit:
chore: remove prerelease version

Commit:
chore: update SonarCloud coverage exclusion

Commit:
chore: update SonarCloud coverage exclusion
